run-tests.sh

```diff
@@ -331,6 +355,22 @@ for arch in $ARCHS; do
                 fi
                 i=$(($i+1))
             done
+            for test in $TESTS_CFGUARD; do
+                file=$test.exe
+                OUT=cmdoutput
+                rm -f $OUT
+                if $RUN $test.exe check_enabled; then
+                    $RUN $test.exe normal_icall
+                    $RUN $test.exe invalid_icall_nocf || [ $? = 2 ]
+                    # We want to check the exit code to be 0xc0000409
+                    # (STATUS_STACK_BUFFER_OVERRUN aka fail fast exception).
+                    # MSYS2 bash does not give us the full 32-bit exit code, so
+                    # we have to rely on cmd.exe to perform the check.
+                    # (This probably doesn't work on WINE, but WINE doesn't
```

Comment:
Nit; I'd prefer to spell Wine with regular upper/lower casing instead of all-uppercase.